problems/accessing-array-values/problem_es.md

```diff
@@ -9,35 +9,35 @@ El número de índice comienza en cero y finaliza en el valor de la propiedad lo
 A continuación, un ejemplo:
 
 ```js
-	var mascotas = ['gato', 'perro', 'rata'];
+	var pets = ['cat', 'dog', 'rat'];
 
-	console.log(mascotas[0]);
+	console.log(pets[0]);
 ```
 
 El código de arriba, imprime el primer elemento del array de `mascotas` - string `gato`
```

Comment:
thanks, I'm going to correct this right now, I've tried to touch the translations the least I could, I've forgotten to change that